discovery/README.md

```diff
@@ -10,8 +10,19 @@ Docker Swarm comes with multiple Discovery backends.
 
 ## Backends
 
+You use a hosted discovery service with Docker Swarm. The service
+maintains a list of IPs in your swam. There are several available
+services, such as `etcd`, `consul` and `zookeeper` depending on what
```

Comment:
@MHBauer you're not moving slow, you're doing excellent work! It's sometimes hard to find the right words, but (as a wise girl once told me): "perfect is the enemy of good" :smile:

So sometimes its better to get things merged, we can always improve in another PR.